packages/grabinfo.rb

```diff
@@ -35,13 +35,13 @@ def self.build
   end
 
   def self.install
-    system "mkdir -p #{CREW_DEST_DIR}#{CREW_PREFIX}/bin"
-    system "mkdir -p #{CREW_DEST_DIR}#{CREW_PREFIX}/man/man1"
-    system "mkdir -p #{CREW_DEST_DIR}#{CREW_PREFIX}/tmp/grabinfo"
-    system "cp grabinfo #{CREW_DEST_DIR}#{CREW_PREFIX}/bin"
-    system "cp GrabInfo.pm #{CREW_DEST_DIR}#{CREW_PREFIX}/bin"
-    system "cp -r GrabInfo/ #{CREW_DEST_DIR}#{CREW_PREFIX}/bin"
-    system "cp grabinfo.1 #{CREW_DEST_DIR}#{CREW_PREFIX}/man/man1"
-    system "touch #{CREW_DEST_DIR}#{CREW_PREFIX}/tmp/grabinfo/grabinfo.lock"
+    system "mkdir -p #{CREW_DEST_PREFIX}/bin"
+    system "mkdir -p #{CREW_DEST_PREFIX}/man/man1"
+    system "mkdir -p #{CREW_DEST_PREFIX}/tmp/grabinfo"
+    system "cp grabinfo #{CREW_DEST_PREFIX}/bin"
+    system "cp GrabInfo.pm #{CREW_DEST_PREFIX}/bin"
+    system "cp -r GrabInfo/ #{CREW_DEST_PREFIX}/bin"
+    system "cp grabinfo.1 #{CREW_DEST_PREFIX}/man/man1"
+    system "touch #{CREW_DEST_PREFIX}/tmp/grabinfo/grabinfo.lock"
```

Comment:
Use `FileUtils ...` instead?

Reply:
As long as rubifying it doesn't change what actually happens, I see no problem with rubyfing a package and not providing updated builds with that new package file.

Reply:
And that package hasn't been updated since 2013, we may be waiting some time for a new release.